Tools/AP_Periph/can.cpp

```diff
@@ -1208,16 +1212,60 @@ static void processTx(void)
         } else {
             // just exit and try again later. If we fail 8 times in a row
             // then start discarding to prevent the pool filling up
-            if (dronecan.tx_fail_count < 8) {
+            if (dronecan.tx_fail_count < 80) {
                 dronecan.tx_fail_count++;
             } else {
+                protocol_stats.tx_errors++;
                 canardPopTxQueue(&dronecan.canard);
             }
             break;
         }
     }
 }
 
+static void update_rx_protocol_stats(int16_t res)
```

Comment:
need to default this off and have a check on board flash size >= 256k

Reply:
Done